README.md

```diff
@@ -245,9 +252,10 @@ interface XRCubeLayer extends XRLayer {
 // Graphics Bindings
 //
 
-dictionary XRWebGLLayerSourceInit {
+dictionary XRLayerInit {
   required unsigned int pixelWidth;
   required unsigned int pixelHeight;
+  boolean framebuffer = true; // should it be false?
```

Comment:
I'm not sure what this is providing? Whether or not you get a framebuffer or texture would be determined by which GraphicsBinding type you're constructing the layers with. As I mentioned to Artem in the previous PR, they would probably be less confusing if they were called something like `XRWebGLFramebufferLayerFactory` and `XRWebGL2TextureLayerFactory` respectively.

Reply:
Does that mean we'll have 4 graphics binding objects? Or could be get by with only 2?(Texture/Framebuffer and switch on what gl flavor is passed in)

Reply:
Only 2. WebGL 1 will only deal with Framebuffers, because it's the only way you can get any antialiasing. WebGL 2 can do both, but if it wants to do Framebuffers instead of textures it should just use the WebGL 1 bindings.

(Okay, I lied: We'll have 3 once WebGPU arrives. It will only do textures.)

Reply:
I see. I was under the impression we needed textures for webgl 1. Maybe @Artyom17  can chime in on what we need.
If it's not needed, it can indeed be simplified.

Reply:
@toji is correct that framebuffer is the only way to render with AA, however, afaik, it is also true for WebGL 2 (there is no direct render-to-texture-multisampling way (yet? we are discussing the corresponding WebGL extension based on EXT_multisampled_render_to_texture GL ES extension at the moment; and that extension could work in WebGL 1 too).
Otherwise, framebufferTexture2D can be still used with WebGL 1, as well as texImage2D (for videos or images). So, I'd say we need textures support for WebGL 1.

Reply:
ok. So maybe we can get by with `XRWebGLFramebufferLayerFactory` and `XRWebGLTextureLayerFactory` but have different behavior depending on the gl context.

For WebGL1, you would get an array of textures and for WebGL2, it would be a texture array

Reply:
@Artyom17 Hm... I didn't realize that you still wanted the ability to use textures with WebGL 1. That complicates things a bit.

Now I'm curious about something: Do you see antialiasing as a necessity for Quad/Cylinder/Equirect layers?

Reply:
@toji so, you are asking if we need a framebuffer support for quad/cyl/equirect? You mean, overall, for either WebGL 1 or 2? I probably can live without framebuffer support for Quad/Cyl/Equirects, hoping that AA WebGL R-2-T extensions are going to be accepted....

Reply:
@Artyom17 is it acceptable that Projection and Cube layers are *only* constructed with framebuffer backends and Quad, Cylinder, Equirect with texture backends?

Reply:
I don't think Cube layers can be represented by a framebuffer. At least not a single one. The way that you render to them is to bind each face as the framebuffer color buffer separately.

Reply:
I could be done with a side-by-side framebuffer similar to the other layers. I don't know if this is done in practice.
Maybe *only* projection layers have framebuffer backend.

Reply:
I've been wondering about that myself, which is part of why I asked about antialiasing. It's worth starting up a separate conversation about.